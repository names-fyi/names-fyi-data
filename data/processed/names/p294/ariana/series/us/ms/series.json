[{"y":1990,"t":6,"m":0,"f":6,"mr":null,"fr":401,"mp":0,"fp":0.00043560331058516046},{"y":1992,"t":8,"m":0,"f":8,"mr":null,"fr":309,"mp":0,"fp":0.0006211180124223603},{"y":1994,"t":12,"m":0,"f":12,"mr":null,"fr":219,"mp":0,"fp":0.0009853025699975368},{"y":1995,"t":9,"m":0,"f":9,"mr":null,"fr":278,"mp":0,"fp":0.000744355305599206},{"y":1996,"t":9,"m":0,"f":9,"mr":null,"fr":284,"mp":0,"fp":0.0007756614668620184},{"y":1997,"t":9,"m":0,"f":9,"mr":null,"fr":275,"mp":0,"fp":0.0007814535035165407},{"y":1998,"t":13,"m":0,"f":13,"mr":null,"fr":223,"mp":0,"fp":0.0010672358591248667},{"y":1999,"t":13,"m":0,"f":13,"mr":null,"fr":213,"mp":0,"fp":0.0011011350160935118},{"y":2000,"t":20,"m":0,"f":20,"mr":null,"fr":147,"mp":0,"fp":0.0016632016632016633},{"y":2001,"t":12,"m":0,"f":12,"mr":null,"fr":230,"mp":0,"fp":0.001034304430270643},{"y":2002,"t":21,"m":0,"f":21,"mr":null,"fr":130,"mp":0,"fp":0.0018561074774615521},{"y":2003,"t":17,"m":0,"f":17,"mr":null,"fr":170,"mp":0,"fp":0.001473519979197365},{"y":2004,"t":15,"m":0,"f":15,"mr":null,"fr":202,"mp":0,"fp":0.001279863481228669},{"y":2005,"t":18,"m":0,"f":18,"mr":null,"fr":156,"mp":0,"fp":0.00155561317085818},{"y":2006,"t":15,"m":0,"f":15,"mr":null,"fr":215,"mp":0,"fp":0.0011996161228406909},{"y":2007,"t":20,"m":0,"f":20,"mr":null,"fr":158,"mp":0,"fp":0.0015696123057604772},{"y":2008,"t":14,"m":0,"f":14,"mr":null,"fr":236,"mp":0,"fp":0.0011218847664075647},{"y":2009,"t":13,"m":0,"f":13,"mr":null,"fr":244,"mp":0,"fp":0.0011059123777116121},{"y":2010,"t":9,"m":0,"f":9,"mr":null,"fr":321,"mp":0,"fp":0.0008192244675040961},{"y":2011,"t":12,"m":0,"f":12,"mr":null,"fr":235,"mp":0,"fp":0.0010788456351703676},{"y":2012,"t":19,"m":0,"f":19,"mr":null,"fr":142,"mp":0,"fp":0.0017873941674506114},{"y":2013,"t":20,"m":0,"f":20,"mr":null,"fr":137,"mp":0,"fp":0.001834357516279923},{"y":2014,"t":26,"m":0,"f":26,"mr":null,"fr":106,"mp":0,"fp":0.002310700319943121},{"y":2015,"t":31,"m":0,"f":31,"mr":null,"fr":79,"mp":0,"fp":0.0027646481762240256},{"y":2016,"t":21,"m":0,"f":21,"mr":null,"fr":129,"mp":0,"fp":0.0018944519621109607},{"y":2017,"t":22,"m":0,"f":22,"mr":null,"fr":126,"mp":0,"fp":0.002018904285583188},{"y":2018,"t":15,"m":0,"f":15,"mr":null,"fr":197,"mp":0,"fp":0.0013988622586962604},{"y":2019,"t":26,"m":0,"f":26,"mr":null,"fr":92,"mp":0,"fp":0.002476426326316792},{"y":2020,"t":22,"m":0,"f":22,"mr":null,"fr":119,"mp":0,"fp":0.002179728524720103},{"y":2021,"t":17,"m":0,"f":17,"mr":null,"fr":152,"mp":0,"fp":0.0017401985873682056},{"y":2022,"t":22,"m":0,"f":22,"mr":null,"fr":108,"mp":0,"fp":0.0022826312512969496}]