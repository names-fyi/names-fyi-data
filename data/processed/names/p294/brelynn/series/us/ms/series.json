[{"y":2011,"t":6,"m":0,"f":6,"mr":null,"fr":491,"mp":0,"fp":0.0005394228175851838},{"y":2012,"t":7,"m":0,"f":7,"mr":null,"fr":395,"mp":0,"fp":0.000658513640639699},{"y":2014,"t":5,"m":0,"f":5,"mr":null,"fr":570,"mp":0,"fp":0.00044436544614290794},{"y":2020,"t":6,"m":0,"f":6,"mr":null,"fr":481,"mp":0,"fp":0.0005944714158327553}]