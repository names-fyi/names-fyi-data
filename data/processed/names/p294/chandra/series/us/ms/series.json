[{"y":1958,"t":5,"m":0,"f":5,"mr":null,"fr":525,"mp":0,"fp":0.0001976206474052409},{"y":1960,"t":5,"m":0,"f":5,"mr":null,"fr":550,"mp":0,"fp":0.0001971453355413611},{"y":1962,"t":5,"m":0,"f":5,"mr":null,"fr":558,"mp":0,"fp":0.00020233085140822273},{"y":1964,"t":6,"m":0,"f":6,"mr":null,"fr":484,"mp":0,"fp":0.0002643987132595955},{"y":1965,"t":11,"m":0,"f":11,"mr":null,"fr":320,"mp":0,"fp":0.0005288970093278199},{"y":1966,"t":14,"m":0,"f":14,"mr":null,"fr":267,"mp":0,"fp":0.000717948717948718},{"y":1967,"t":28,"m":0,"f":28,"mr":null,"fr":153,"mp":0,"fp":0.001498367849307005},{"y":1968,"t":21,"m":0,"f":21,"mr":null,"fr":182,"mp":0,"fp":0.0011642734379331374},{"y":1969,"t":28,"m":0,"f":28,"mr":null,"fr":151,"mp":0,"fp":0.0015045674368619022},{"y":1970,"t":28,"m":0,"f":28,"mr":null,"fr":156,"mp":0,"fp":0.0014902336473468518},{"y":1971,"t":29,"m":0,"f":29,"mr":null,"fr":147,"mp":0,"fp":0.0015549597855227882},{"y":1972,"t":20,"m":0,"f":20,"mr":null,"fr":197,"mp":0,"fp":0.0011536021226279055},{"y":1973,"t":20,"m":0,"f":20,"mr":null,"fr":185,"mp":0,"fp":0.0012188433176915107},{"y":1974,"t":18,"m":0,"f":18,"mr":null,"fr":202,"mp":0,"fp":0.0011202389843166542},{"y":1975,"t":16,"m":0,"f":16,"mr":null,"fr":220,"mp":0,"fp":0.0010286081645773063},{"y":1976,"t":16,"m":0,"f":16,"mr":null,"fr":220,"mp":0,"fp":0.0010534632604687912},{"y":1977,"t":22,"m":0,"f":22,"mr":null,"fr":170,"mp":0,"fp":0.0013915243516761542},{"y":1978,"t":23,"m":0,"f":23,"mr":null,"fr":157,"mp":0,"fp":0.0014998369742419302},{"y":1979,"t":16,"m":0,"f":16,"mr":null,"fr":218,"mp":0,"fp":0.0010264964393404761},{"y":1980,"t":20,"m":0,"f":20,"mr":null,"fr":180,"mp":0,"fp":0.0012183235867446393},{"y":1981,"t":13,"m":0,"f":13,"mr":null,"fr":242,"mp":0,"fp":0.0008209662140827281},{"y":1982,"t":12,"m":0,"f":12,"mr":null,"fr":264,"mp":0,"fp":0.0007507977225802415},{"y":1983,"t":11,"m":0,"f":11,"mr":null,"fr":279,"mp":0,"fp":0.000721642721249098},{"y":1984,"t":8,"m":0,"f":8,"mr":null,"fr":345,"mp":0,"fp":0.0005318088147311041},{"y":1986,"t":6,"m":0,"f":6,"mr":null,"fr":405,"mp":0,"fp":0.00043636363636363637},{"y":1987,"t":6,"m":0,"f":6,"mr":null,"fr":396,"mp":0,"fp":0.0004510599909788002},{"y":1988,"t":8,"m":0,"f":8,"mr":null,"fr":324,"mp":0,"fp":0.0005940888162780336},{"y":1989,"t":7,"m":0,"f":7,"mr":null,"fr":348,"mp":0,"fp":0.0005111727763984227}]