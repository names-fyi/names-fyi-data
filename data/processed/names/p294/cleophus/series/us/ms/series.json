[{"y":1939,"t":6,"m":6,"f":0,"mr":344,"fr":null,"mp":0.0002842524161455372,"fp":0},{"y":1955,"t":6,"m":6,"f":0,"mr":403,"fr":null,"mp":0.00020749757919490938,"fp":0},{"y":1956,"t":7,"m":7,"f":0,"mr":372,"fr":null,"mp":0.00024532137099600476,"fp":0},{"y":1958,"t":7,"m":7,"f":0,"mr":368,"fr":null,"mp":0.0002590769458529183,"fp":0},{"y":1961,"t":5,"m":5,"f":0,"mr":431,"fr":null,"mp":0.0001852949896234806,"fp":0}]