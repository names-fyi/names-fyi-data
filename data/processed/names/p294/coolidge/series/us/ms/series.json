[{"y":1924,"t":6,"m":6,"f":0,"mr":353,"fr":null,"mp":0.00032481593763534,"fp":0},{"y":1925,"t":6,"m":6,"f":0,"mr":349,"fr":null,"mp":0.00031423483816905834,"fp":0}]