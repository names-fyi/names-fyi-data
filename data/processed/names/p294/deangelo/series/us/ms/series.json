[{"y":1984,"t":6,"m":6,"f":0,"mr":364,"fr":null,"mp":0.0003362474781439139,"fp":0},{"y":1985,"t":9,"m":9,"f":0,"mr":292,"fr":null,"mp":0.0005182242183451373,"fp":0},{"y":1986,"t":12,"m":12,"f":0,"mr":234,"fr":null,"mp":0.0007029465174857946,"fp":0},{"y":1987,"t":13,"m":13,"f":0,"mr":214,"fr":null,"mp":0.0007940386024920597,"fp":0},{"y":1988,"t":11,"m":11,"f":0,"mr":241,"fr":null,"mp":0.0006712638066760237,"fp":0},{"y":1989,"t":17,"m":17,"f":0,"mr":196,"fr":null,"mp":0.0009903297215425842,"fp":0},{"y":1990,"t":16,"m":16,"f":0,"mr":208,"fr":null,"mp":0.0009331078322738672,"fp":0},{"y":1991,"t":19,"m":19,"f":0,"mr":188,"fr":null,"mp":0.0011345315578909655,"fp":0},{"y":1992,"t":11,"m":11,"f":0,"mr":265,"fr":null,"mp":0.0006713047723666544,"fp":0},{"y":1993,"t":15,"m":15,"f":0,"mr":222,"fr":null,"mp":0.0009468501451836889,"fp":0},{"y":1994,"t":9,"m":9,"f":0,"mr":308,"fr":null,"mp":0.0005748961992973491,"fp":0},{"y":1995,"t":12,"m":12,"f":0,"mr":263,"fr":null,"mp":0.0007971303308090873,"fp":0},{"y":1996,"t":19,"m":19,"f":0,"mr":170,"fr":null,"mp":0.0012547051442910915,"fp":0},{"y":1997,"t":8,"m":8,"f":0,"mr":359,"fr":null,"mp":0.0005347236147316356,"fp":0},{"y":1998,"t":7,"m":7,"f":0,"mr":382,"fr":null,"mp":0.00045641259698767686,"fp":0},{"y":1999,"t":6,"m":6,"f":0,"mr":448,"fr":null,"mp":0.00039452919516044186,"fp":0},{"y":2000,"t":8,"m":8,"f":0,"mr":368,"fr":null,"mp":0.0005143702179643798,"fp":0},{"y":2001,"t":6,"m":6,"f":0,"mr":449,"fr":null,"mp":0.0004087471898630697,"fp":0},{"y":2006,"t":7,"m":7,"f":0,"mr":454,"fr":null,"mp":0.0004365995135033992,"fp":0},{"y":2008,"t":8,"m":8,"f":0,"mr":405,"fr":null,"mp":0.000508808751510526,"fp":0},{"y":2011,"t":8,"m":8,"f":0,"mr":373,"fr":null,"mp":0.0005761198329252485,"fp":0},{"y":2014,"t":5,"m":5,"f":0,"mr":535,"fr":null,"mp":0.0003633456870866943,"fp":0},{"y":2018,"t":5,"m":5,"f":0,"mr":547,"fr":null,"mp":0.0003858620157431702,"fp":0}]