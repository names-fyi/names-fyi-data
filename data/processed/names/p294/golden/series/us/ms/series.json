[{"y":1917,"t":5,"m":0,"f":5,"mr":null,"fr":438,"mp":0,"fp":0.0002840909090909091},{"y":1918,"t":6,"m":0,"f":6,"mr":null,"fr":406,"mp":0,"fp":0.00031104199066874026}]